src/minter-did/src/init.rs

```diff
@@ -64,12 +65,95 @@ impl Default for OperationPricing {
     }
 }
 
+impl OperationPricing {
+    pub const STORABLE_BYTE_SIZE: usize = size_of::<u32>() * 6;
+}
+
+
 impl Storable for OperationPricing {
     fn to_bytes(&self) -> Cow<'_, [u8]> {
-        codec::encode(self).into()
+        let mut buf = Vec::with_capacity(Self::STORABLE_BYTE_SIZE);
+        buf.extend_from_slice(&self.evmc_notification.to_be_bytes());
```

Comment:
The code would look better if `encode_u32` and `decode_u32` functions were extracted

Reply:
It seems a little over-engineered to me, could you propose how you would do it?

Reply:
Refactored, and now it looks much better. Let me know

Reply:
Looks much better to me